### ISSUE TITLE
Vault name based local database sync

### ISSUE THREAD

**Issue by ema-s** [vrtmrz/obsidian-livesync#92]
Hi, i am appreciating your plugin very much, however i have an issue with local database sync. 
If i have two vaults with the same name (but different path) the plugin will sync them (even with live sync option off). 
This is giving me issues because if i don't pay attention to naming vaults differently i will get different projects messed up together or have data overwritten on older vaults. 
I would ask that the local database sync does not rely on vault name alone, or that the local db sync could be turned off.

thank you.

**Comment by vrtmrz** [comment]
Thank you for notifying this! 
It was a designed specification for moving the vault to another folder, but you're also right. This situation can't be avoidable in current behaviour. 
I think it has to be improved.

**Comment by vrtmrz** [comment]
@ema-s
Implemented on v0.12.2. Could you please try it?
(Of course, we still need attention to installing LiveSync. Please place `redflag.md` on the root folder to avoid unwanted synchronization until you have configured LiveSync.)

**Comment by ema-s** [comment]
Thank you very much, that was super fast! will try the new version and let you know :)

**Comment by ema-s** [comment]
ok using the redflag.md i get exactly what i wanted and i can have them not sync by default! thank you

**Comment by vrtmrz** [comment]
I was relieved to hear that! 
Did the database suffix also work well?

I forgot to document it. Please set something (I had testing this as setting ‘-2’ to the second vault.) to be unique If we want to use two or more vaults which having the same name.

**Comment by ema-s** [comment]
yes i tried using 1 and 2 as local db names for two vaults with same name and they did not sync :) 
could you possibly make a toggle setting that would allow to turn off local db sync, the same way as live-sync can be toggled off? which is basically what the redflag.md does but having it as a setting?

**Comment by vrtmrz** [comment]
@ema-s 
Thank you for your testing! We made it !

Unfortunately, I have no other smart idea to implement this feature. If we could get the whole vault names in the plugin, we would be able to ask the suffix on the second setup...

`redflag.md` turns off every feature of LiveSync before everything gets started, even though we make a misconfiguration or the configuration file has been corrupted. Maybe troublesome if it is within the LiveSync's configuration.

It may be a slightly different idea, but the feature we need is `Suspend and make redflag.md immediately`, `Resume from redflag.md`; with saving the synchronisation setting into redflag or reloading them from redflag.
May I have your opinion? And I hope to ask you suitable command names.
